public/components/ui_action_context_menu.tsx

```diff
@@ -148,7 +157,15 @@ export const ActionContextMenu = (props: Props) => {
       anchorPosition="downRight"
       closePopover={() => setOpen(false)}
     >
-      <EuiContextMenu size="s" initialPanelId={'mainMenu'} panels={panels.value} />
+      {showGenerateVisualization ? (
+        <EuiContextMenu size="s" initialPanelId={'mainMenu'} panels={panels.value} />
+      ) : (
+        <EuiToolTip content="This button be disabled because of No Results/Error.">
+          <div style={{ pointerEvents: 'none' }}>
+            <EuiContextMenu size="s" initialPanelId={'mainMenu'} panels={panels.value} />
+          </div>
+        </EuiToolTip>
```

Comment:
even through we set the style of pointers, what will happen if user still click on the menu?